substrate/frame/nomination-pools/src/lib.rs

```diff
@@ -3092,9 +3098,11 @@ pub mod pallet {
 	}
 
 	#[pallet::hooks]
-	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
+	impl<T: Config> Hooks<frame_system::pallet_prelude::BlockNumberFor<T>> for Pallet<T> {
```

Comment:
personally I would have introduced a different type alias to avoid the name collision with the `frame_system::pallet_prelude::BlockNumberFor`.
But I don't mind any implementation.

Reply:
i have done as suggested

Reply:
I was thinking the opposite like keeping `BlockNumberFor` for the system one, and having maybe `ProvidedBlockNumberFor` or `PalletBlockNumberFor`.
But this is also good to me.